scripts/retract_registrations.py

```diff
@@ -0,0 +1,36 @@
+"""Script for retracting pending retractions that are more than 48 hours old."""
+
+import datetime
+import logging
+import sys
+
+from modularodm import Q
+
+from website import models, settings
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
```

Comment:
Again, if this param is only for logging consider dropping it from the signature.